[OKTA-730760] Content issues in guide (#4849)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md

When user schema discovery is enabled for an app integration, admins can't add attributes to the app user profile in the Profile Editor in the Admin Console. This option is disabled because the SCIM server determines the contents of the app user profile between Okta and the downstream app. Okta gathers profile elements from the SCIM server to dynamically build the app user profile.

The SCIM server should be updated to maintain parity with the downstream app. User schema discovery allows the app provider to determine the contents of the app user profile.

### Create an app that supports entitlements and user schema discovery

1. [Create a SCIM app integration with entitlement management](https://help.okta.com/okta_help.htm?type=oie&id=ext-aiw-scim-entitlements).
1. Ensure that your SCIM 2.0 server exposes the following endpoints:
    - `/ResourceTypes`
    - Any resource-specific endpoints for entitlements (for example: `/Licenses`, `/Roles`, and so on)
1. Optional. If your app requires custom entitlement, role, or user schema extensions, expose the following endpoint:
    - `/Schemas`
1. Configure your app in Okta.

### User operations

> Create, update, and import user operations are handled using [SCIM User operations](/docs/reference/scim/scim-20/#scim-user-operations). The user object should include any Entitlements, Roles, and custom user schema extensions that your app requires. See [User with entitlements and roles](#user-with-entitlements-and-roles) for an example.

Reviewer: ```suggestion
Create, update, and import user operations are handled using [SCIM User operations](/docs/reference/scim/scim-20/#scim-user-operations). The user object should include any entitlements, roles, and custom user schema extensions that your app requires. See [User with entitlements and roles](#user-with-entitlements-and-roles) for an example.
```

### Endpoints for user schema discovery

#### /ResourceTypes

Okta calls this endpoint to gather a list of available resources with any associated schemas and schema extension URNs.

Common resource types include the following:

- `Users`
- `Groups`
- `Entitlements`
- `Roles`

Okta doesn't offer custom handling for groups.

> To expose entitlements and roles, you must create a corresponding `ResourceType` for each of these entities. See [ResourceTypes](#resourcetypes) for an implementation example.

Reviewer: ```suggestion
To expose entitlements and roles, create a corresponding `ResourceType` for each of these entities. See [ResourceTypes](#resourcetypes) for an implementation example.
```

    - Role: `urn:okta:scim:schemas:core:1.0:Role`
    - Entitlement: `urn:okta:scim:schemas:core:1.0:Entitlement`
    - The schema for users is the standard: `urn:ietf:params:scim:schemas:core:2.0:User`
* Schema extensions: List any extensions required for other properties. Generally, entitlements and roles don't need to have extensions, while it's common for users to have highly customized extensions. For example, a user might have a schema extension to store a custom attribute for a particular app.

> The default location for `ResourceTypes` definitions is `BaseURL/scim/v2/ResourceTypes`. For example, if your server is hosted on `https://example.com`, then the available ResourceTypes can be retrieved from `https://example.com/scim/v2/ResourceTypes`.

Reviewer: ```suggestion
The default location for `ResourceTypes` definitions is `BaseURL/scim/v2/ResourceTypes`. For example, if your server is hosted on `https://example.com`, then you can retrieve the available `ResourceTypes` from `https://example.com/scim/v2/ResourceTypes`.
```
